README.md

```diff
@@ -62,9 +79,54 @@ Alternatively, create a new config file named `config.json`,
             "commit": "6bd6b76d1156e20e45d1016f355d154141c7e5b9",
             "relative_root": "contracts"
         }
+    },
+    "raise_exception": true,
+    "bytecode_comparison": {
+        "hardhat_config_name": "holesky_hardhat.config.js",
+        "constructor_calldata": {
+            "0x28FAB2059C713A7F9D8c86Db49f9bb0e96Af1ef8": "000000000000000000000000ab89ed3d8f31bcf8bb7de53f02084d1e6f043d34000000000000000000000000e92329ec7ddb11d25e25b3c21eebf11f15eb325d00000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000000"
+        },
+        "constructor_args": {
+            "0x28FAB2059C713A7F9D8c86Db49f9bb0e96Af1ef8": [
+                "0xab89ED3D8f31bcF8BB7De53F02084d1e6F043D34",
+                "0xE92329EC7ddB11D25e25b3c21eeBf11f15eB325d",
+                ""
+            ],
+            "0xDba5Ad530425bb1b14EECD76F1b4a517780de537": [
+                [
+                    "0x4E97A3972ce8511D87F334dA17a2C332542a5246",
+                    "0x045dd46212A178428c088573A7d102B9d89a022A",
+                    "0xE73a3602b99f1f913e72F8bdcBC235e206794Ac8",
+                    "0x072f72BE3AcFE2c52715829F2CD9061A6C8fF019",
+                    "0x3F1c547b21f65e10480dE3ad8E19fAAC46C95034",
+                    "0xF0d576c7d934bBeCc68FE15F1c5DAF98ea2B78bb",
+                    "0x072f72BE3AcFE2c52715829F2CD9061A6C8fF019",
+                    "0x4E46BD7147ccf666E1d73A3A456fC7a68de82eCA",
+                    "0xd6EbF043D30A7fe46D1Db32BA90a0A51207FE229",
+                    "0xE92329EC7ddB11D25e25b3c21eeBf11f15eB325d",
+                    "0xffDDF7025410412deaa05E3E1cE68FE53208afcb",
+                    "0xc7cc160b58F8Bb0baC94b80847E2CF2800565C50",
+                    "0xF0179dEC45a37423EAD4FaD5fCb136197872EAd9",
+                    "0xC01fC1F2787687Bc656EAc0356ba9Db6e6b7afb7"
+                ]
+            ]
+        }
     }
 }
 ```
+use `holesky_hardhat.config.js` from `config.json` or create near the diffyscan.py own Hardhat config `hardhat_config.js`: 
```

Comment:
Suppose I installed diffyscan using the command from the Install section of README:
```
pipx install git+https://github.com/lidofinance/diffyscan
```

Where should I place this, then?

Reply:
the readme has been updated in this part